### PR TITLE
[WIP] Update dependencies 4/22

### PR DESCRIPTION
**Pull request by LindsayYoung**
Fixes issue #16 

locally run:
`composer update`

I got rid of my database volumes and rebuild my images, so you might need to do that.

Changes proposed in this pull request:
- Updated core
- Updated devel, and move that to dev only (It gets run by default but we configure it not to run in prod in the future as part of . #15 )

I did add the default file changes, since they already exist on this branch.

I did local testing on content creation and experience any upgrade problems, 

Caveat - I have to merge in PR #7  locally to get things to run

### CODE REVIEW

**Comment by LindsayYoung** [comment]
I am seeing a piece from #7, I can remove that now

**Comment by LindsayYoung** [comment]
I had issues merging this on another branch over in https://github.com/18F/crt-portal so I don't want to waste anyone's time if there is an issue. Until I know this branch is not the problem.

File: web/.htaccess [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #
 # Apache/PHP/Drupal settings:
 #
-RewriteRule ^s3fs-(css|js)/(.*)$ http://S3_BUCKET.s3-S3_REGION.amazonaws.com/public/$2 [P]
```
Comment (fureigh) [inline web/.htaccess]: @LindsayYoung We need this line; see https://github.com/18F/cf-ex-drupal8/pull/7#discussion_r274376209.

Comment (LindsayYoung) [reply web/.htaccess]: Thanks for the explanation!

File: composer.json [hunk 2]
```diff
@@ -87,5 +86,8 @@
                 "Make menu items export properly": "web/patches/menu_link_uuid_nid-2670954-reroll.patch"
             }
         }
+    },
```
Comment (fureigh) [inline composer.json]: @LindsayYoung Would you be willing to move this piece to a separate PR? In the interest of separating concerns more clearly.

File: web/.htaccess (hunk 1 above)
Comment (fureigh) [reply web/.htaccess]: You're welcome! If you remove this change from the PR, I believe we'll be good to go.

**Comment by LindsayYoung** [comment]
@fureigh thank you for your review and your patience! 

I made the changes, but I still can't get this running locally (without PR 7) to make sure it works.

**Comment by markdboyd** [comment]
Stale and now handled by dependabot
